app/scripts/lib/transform.js

```diff
@@ -33,13 +51,21 @@ define(function (require, exports, module) {
     transformUsingSchema (data, schema, Errors) {
       var result = Vat.validate(data, schema);
       var error = result.error;
-      if (error instanceof ReferenceError) {
-        throw Errors.toMissingParameterError(error.key);
-      } else if (error instanceof TypeError) {
-        throw Errors.toInvalidParameterError(error.key);
-      }
+      var emptyEmail = checkEmptyEmail(data);
```

Comment:
We try to keep this file as generic as possible and define specifics in validation schema in the [relier models](https://github.com/mozilla/fxa-content-server/blob/c1cb23dd64011e582bb32a1395fd334987c8f9ea/app/scripts/models/reliers/relier.js#L33). I think for this case it should be sufficient to change [this line](https://github.com/mozilla/fxa-content-server/blob/c1cb23dd64011e582bb32a1395fd334987c8f9ea/app/scripts/models/reliers/relier.js#L54) to be `Vat.string().allow('')`

Reply:
@shane-tomlinson I was also thinking the same, it was a quick fix for this. Should I only change Vat.string().min(1) to Vat.string().allow('') ?

Reply:
I got it @shane-tomlinson !